frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/InsideHearingMainPage.js

```diff
@@ -277,7 +277,7 @@ const InsideHearingMainPage = () => {
     }
   };
 
-  const attendanceCount = useMemo(() => hearing?.attendees?.filter((attendee) => attendee.wasPresent).length || 0, [hearing]);
+  const attendanceCount = useMemo(() => hearing?.attendees?.filter((attendee) => attendee.wasPresent)?.length || 0, [hearing]);
```

Comment:
`filter()` will always return an array, please don't flood conditional chaining operations since the bundler unpacks them as individual isNull and isUndefined checks

Reply:
addressed